From b29826379068a05cdd42ba6e956d17e4d6681c7b Mon Sep 17 00:00:00 2001
From: Federico Mena Quintero <federico@gnome.org>
Date: Tue, 23 Nov 2021 11:18:51 -0600
Subject: [PATCH] Fix function prototype

device_remove_datum already implicitly casts its cb to a
AtspiDeviceListenerSimpleCB, which takes a const *event.

Signed-off-by: Martin Jansa <martin.jansa@gmail.com>
Upstream-Status: Backport [v2.43.92 https://github.com/GNOME/at-spi2-core/commit/1e91fc4cff2080696be914e26f4cdf0bf32d1550]
---
 atspi/atspi-device-listener.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/atspi/atspi-device-listener.c b/atspi/atspi-device-listener.c
index 69f77d1..9776ebd 100644
--- a/atspi/atspi-device-listener.c
+++ b/atspi/atspi-device-listener.c
@@ -53,7 +53,7 @@ device_event_handler_new (AtspiDeviceListenerCB callback,
 }
 
 static gboolean
-device_remove_datum (const AtspiDeviceEvent *event, void *user_data)
+device_remove_datum (AtspiDeviceEvent *event, void *user_data)
 {
   AtspiDeviceListenerSimpleCB cb = user_data;
   return cb (event);
